src/main/webapp/i18n/de/exercise.json

```
@@ -130,6 +130,9 @@
             "personalRepositoryClone": "Dein persönliches Repository wurde eingerichtet. Klicke auf den Button <i>Repository klonen</i>!",
             "personalRepositoryOnline": "Dein persönliches Repository wurde eingerichtet. Klicke auf den Button <i>Programmiereditor öffnen</i>!",
             "resumeProgrammingExercise": "Die Aufgabe wurde wieder aufgenommen. Du kannst nun weiterarbeiten!",
+            "feedbackRequestSent": "Deine Feedback anfrage wurde gesendet",
+            "feedbackRequestAlreadySent": "Deine Feedback anfrage wurde bereits gesendet",
+            "lockRepositoryWarning": "Dein Repository wird gelocked. Du kannst weiter arbeiten wenn ein feedback eingereicht wurde",
```

```suggestion
            "lockRepositoryWarning": "Dein Repository wird gesperrt. Du kannst erst weiterarbeiten wenn ein Tutor die Feedbackanfrage beantwortet hat.",
```
Possibly similar changes are applicable for the English translation.